packages/jest-cli/src/watch.js

```diff
@@ -93,7 +75,7 @@ const watch = (
         // and prevent test runs from the previous run.
         testWatcher = new TestWatcher({isWatchMode: true});
         if (displayHelp) {
-          console.log(usage(argv, hasSnapshotFailure));
+          pipe.write(usage(argv, hasSnapshotFailure));
```

Comment:
I changed this and https://github.com/facebook/jest/pull/2362/commits/7909aebe08f214d978d69ce6e2425be824d6e529#diff-d8a3d0a3a8b42ab87b18835ad8f7cb46L177 to use `pipe.write` instead of `console.log` for testability purposes, but I'm unsure if this has any impact in on how things are printed out that I'm unaware of... as far as I can tell it works just fine